.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -197,7 +197,7 @@ jobs:
 
     - name: Install Dependencies (Python)
       run: |
-        python${{ matrix.python_version }} -m pip install wheel numpy pybind11 PyYAML
+        python${{ matrix.python_version }} -m pip install wheel numpy pybind11 PyYAML scipy
```

Comment:
Thanks for the details! This is a build time dependency, I think adding it to the setup.py is useless.

Reply:
I think it was needed here because `$(PYTHON) $(MK_DIR)/setup.py bdist_wheel` does not make use of the toml file.

Reply:
> I think it was needed here because $(PYTHON) $(MK_DIR)/setup.py bdist_wheel does not make use of the toml file.

Oh I see, fair enough 👍 

> This is a build time dependency, I think adding it to the setup.py is useless.

Well I think this is still useful in this case since it is _also_ a runtime dependency right? The user needs to have scipy installed when they run catalyst (which they will have installed because jax depends on it but I think being explicit it still good).